### PR TITLE
Add user DBUS module covering the rootpw command

### PR DESCRIPTION
**Pull request by M4rtinK**
The first commit adds the DBUS module & tests.
The second commit switches kickstart.py to use the user DBUS module for rootpw.
The third commit switches the UI to use the DBUS module for rootpw.

**TODO:** More testing! This has not yet been tested a whole lot aside from unit tests.

### CODE REVIEW

File: pyanaconda/modules/user/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# modules/user/Makefile.am for anaconda
```
Comment (poncovka) [inline pyanaconda/modules/user/Makefile.am]: Please, remove this line.

File: pyanaconda/modules/user/user.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        self._root_password = root_password
+        self._root_password_is_crypted = crypted
+        self.root_password_set_changed.emit()
+        log.debug("Root password set.")
+
+    def clear_root_password(self):
+        """Clear any set root password."""
+        self._root_password = ""
+        self._root_password_is_crypted = False
+        self.root_password_set_changed.emit()
```
Comment (poncovka) [inline pyanaconda/modules/user/user.py]: Please, keep the name of the signal consistent with the name of the property. In this case, it should be `root_password_is_set_changed`.

File: pyanaconda/modules/user/user_kickstart.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#
```
Comment (poncovka) [inline pyanaconda/modules/user/user_kickstart.py]: Please, rename this file to `kickstart.py`.

File: pyanaconda/modules/user/Makefile.am (hunk 1 above)
Comment (M4rtinK) [reply pyanaconda/modules/user/Makefile.am]: Fixed.

File: pyanaconda/modules/user/user.py (hunk 2 above)
Comment (M4rtinK) [reply pyanaconda/modules/user/user.py]: Good point! Fixed.

File: pyanaconda/modules/user/user_kickstart.py (hunk 3 above)
Comment (M4rtinK) [reply pyanaconda/modules/user/user_kickstart.py]: Renamed.

**Pull request merged**